Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM europe-docker.pkg.dev/kyma-project/prod/external/golang:1.22.2-alpine3.19 as builder
+FROM golang:1.23.1-alpine3.20 as builder
```

Comment:
Is there a reason for not using the same image as on telemetry-manager? Namely:

`europe-docker.pkg.dev/kyma-project/prod/external/library/golang:1.23.1-alpine3.20`

Reply:
See: https://github.com/kyma-project/telemetry-manager/pull/1446/files

Reply:
fixed